blog/_posts/2016-08-18-coaching-companies-part-1.md

```diff
@@ -0,0 +1,152 @@
+---
+title: RGSoC in a Coaching Company - Part 1
+layout: post
+created_at: Thu August 18 2016
+permalink: blog/2016-08-18-coaching-companies-part-1
+current: blog
+author: Ana Sofia Pinho
+twitter: RailsGirlsSoC
+categories:
```

Comment:
can you add a category here? I think `news` is enough, unless you'd like to create a special new category for it as well.